classes/Models/User.php

```diff
@@ -264,6 +264,10 @@ public static function getUsers( array $args ) {
 			$where_clause .= $wpdb->prepare( ' AND _user.display_name LIKE %s', "%{$wpdb->esc_like( $args['search'] )}%" );
 		}
 
+		if ( ! empty( $args['employee_status'] ) ) {
+			$where_clause .= $wpdb->prepare( ' AND _employment.employment_status = %s', "{$wpdb->esc_like( $args['employee_status'] )}" );
```

Comment:
`$wpdb->esc_like` is not supposed to be used in this case as it is direct equal comparison, not `LIKE` condition.

Reply:
removed $wpdb->esc_like from query